src/sweetalert2.scss

```diff
@@ -807,7 +810,7 @@ body {
     width: $swal2-progress-steps-distance;
     height: .4em;
     margin: 0 -1px;
-    background: $blue;
+    background: $swal2-progress-steps-active-step-color;
```

Comment:
The ending of a variable should match the CSS property, in this case it would be:

```diff
- $swal2-progress-steps-active-step-color
+ $swal2-progress-steps-active-step-backgroud
```

Reply:
done in https://github.com/sweetalert2/sweetalert2/pull/1411/commits/3426355c9dd9bb286ade28e808c3718216a7aced